tests/safety/test_subaru.py

```diff
@@ -24,12 +50,23 @@ class TestSubaruSafety(common.PandaSafetyTest, common.DriverTorqueSteeringSafety
   DRIVER_TORQUE_ALLOWANCE = 60
   DRIVER_TORQUE_FACTOR = 50
 
-  ALT_BUS = 0
+  ALT_BUS = SUBARU_MAIN_BUS
 
+  MAX_RATE_UP = 50
+  MAX_RATE_DOWN = 70
+  MAX_TORQUE = 2047
+
+  @classmethod
+  def setUpClass(cls):
+    if cls.__name__.endswith("Base"):
+      cls.packer = None
+      cls.safety = None
+      raise unittest.SkipTest
```

Comment:
PandaSafetyTest already does this for you, but hmm `TorqueSteeringSafetyTestBase` overrides it. Want to clean all that interaction up in another PR?

https://github.com/commaai/panda/blob/6b9b448259d1790ed8e18abde09ee72446996b3a/tests/safety/common.py#L689-L693